configs/hive.yaml

```diff
@@ -0,0 +1,95 @@
+# Mainnet config
+
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
```

Comment:
I did a small modification in `setup.py` for this change to work: It was failing because there is no hive kzg setup, so I added `EQUIVALENT_KZG_SETUPS` map to re-utilize the existing mainnet kzg setup.